kie-drools-wb/kie-drools-wb-distribution-wars/src/main/assembly/component-kie-drools-wb-tomcat-7-common.xml

```diff
@@ -98,8 +100,6 @@
           <!-- Errai -->
           <exclude>WEB-INF/lib/errai-jboss-as-support-*.jar</exclude>
           <exclude>WEB-INF/lib/errai-cdi-server-*.jar</exclude>
```

Comment:
@csadilek thanks, that was my (latest) understanding as well. Will remove the server exclusion and add the client one (and hope that it will fix the issue :))